### PR TITLE
Modified the caption to Latsalomao-caption branch

### PR DESCRIPTION
**Pull request by Latsalomao**
Close #2

### CODE REVIEW

File: _slides/03-Latsalomao.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello!, my name is Tamatoa
```
Comment (vmcherukuri) [inline _slides/03-Latsalomao.md:6]: Reviewed

Comment (vmcherukuri) [inline _slides/03-Latsalomao.md:6]: d

**Review by pranav0903 — COMMENTED** [review]
.

File: _slides/03-Latsalomao.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (vmcherukuri) [inline _slides/03-Latsalomao.md:3]:
```suggestion
title: "Welcome to our Latsalomao slide!"
```

**Review by vmcherukuri — CHANGES_REQUESTED** [review]
Summary of my review

**Review by beardofedu — APPROVED** [review]

**Review by vmcherukuri — APPROVED** [review]

**Pull request merged**